examples/usecases/transformers-next-item-prediction.ipynb

```diff
@@ -0,0 +1,910 @@
+{
```

Comment:
<p>can you explain here some args with one sentence?  what's this <code>pre=mm.ReplaceMaskedEmbeddings()</code> doing for example? Can you pls explain that under the hood this is doing masked language modeling and masking the items randomly? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/853/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>explanation added</p>